service_container.rst

```diff
@@ -366,8 +366,6 @@ made. To do that, you create a new class::
                     'Someone just updated the site. We told them: '.$happyMessage
                 );
             $this->mailer->send($message);
```

Comment:
What about returning `return $this->mailer->send($message) > 0;` and only adding a success flash message if it was indeed succesfull ?